translator/translate/otel/pipeline/host/translator.go

```diff
@@ -65,6 +66,28 @@ func (t translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators,
 		translators.Processors.Set(ec2taggerprocessor.NewTranslator())
 	}
 
+	switch v := conf.Get(common.ConfigKey(common.MetricsKey, common.MetricsCollectedKey, common.JmxKey)).(type) {
```

Comment:
This should not be in the host pipeline. The JMX pipeline needs to be completely separate due to the filters.